src/app/give-help/material-aid-form/material-aid-form.component.html

```diff
@@ -14,7 +14,11 @@
             required
             name="title"
             [(ngModel)]="data.title"
+            #title="ngModel"
+            appOfferTitleValidate
           />
+          <mat-error *ngIf="title.errors?.['required']">Pole jest wymagane</mat-error>
+          <mat-error *ngIf="title.errors?.['offerTitleInvalid']">Dozwolone znaki to [a-zA-Z]</mat-error>
```

Comment:
moze lepiej stworzyć w shared component  field-error/custom-error w input przekazać ngModel, w tym przypadku title
a w środku wylistować wszystkie dostępne errory:

<mat-error *ngIf="control.hasError('required')">Pole jest wymagane</mat-error>
<mat-error *ngIf="control.hasError('offerTitleInvalid')">Dozwolone znaki to [a-zA-Z]</mat-error>
itd....

Reply:
Dokładnie coś takiego planuję zrobić.